rpa-home/modules/ROOT/pages/automation-tutorial-composer.adoc

```diff
@@ -0,0 +1,67 @@
+= Step 8: Integrate the Automation with Composer Flows
+
+After successfully publishing the process automation, you can connect your MuleSoft RPA account with MuleSoft Composer and trigger the automated process from a Composer flow. During this step you create a new MuleSoft Composer flow, configure a trigger to start the execution, connect your Composer account with your MuleSoft RPA account, and trigger the execution of an RPA Process. 
+
+You can configure different triggers to start a Composer flow, including interactions with Salesforce and other systems. However, this tutorial uses the Scheduler event because it is a simple way to illustrate the process and start the execution of the flow without having to configure connections with other systems. 
```

Comment:
```suggestion
Different triggers can start a Composer flow, including interactions with Salesforce and other systems. This tutorial uses the Scheduler event because it is a simple way to start the execution of the flow without configuring connections to other systems. 
```